railties/lib/rails/test_unit/runner.rb

```diff
@@ -59,6 +57,27 @@ def compose_filter(runnable, filter)
         end
 
         private
+          def test_paths(argv)
+            filtered_test_paths(argv) || default_test_paths
+          end
+
+          def default_test_paths
+            Rake::FileList[default_test_glob].exclude(default_test_exclude_glob)
+          end
+
+          def default_test_glob
+            ENV.fetch("DEFAULT_TEST", DEFAULT_TEST_GLOB)
+          end
+
+          def default_test_exclude_glob
+            ENV.fetch("DEFAULT_TEST_EXCLUDE", DEFAULT_TEST_EXCLUDE_GLOB)
+          end
+
+          def filtered_test_paths(argv)
+            patterns = extract_filters(argv)
+            Rake::FileList[patterns] if patterns.any?
```

Comment:
`#presence` not available since you can't count on Rails being loaded

Reply:
We actually need to add a missing `require "active_support"` up top (we require callers to load the top level file to spare internal require calls). Then `presence` is available I think.